crates/runtime/src/lib.rs

```diff
@@ -124,9 +141,18 @@ impl<'a> HintProcessorLogic for StarknetRuntime<'a> {
         })) = maybe_extended_hint
         {
             let selector = parse_selector(selector)?;
-            return Err(HintError::CustomHint(
-                format!("Cheatcode `{selector}` is not supported in this runtime").into(),
-            ));
+
+            let mut base_error =
+                format!("Cheatcode `{selector}` is not supported in this runtime\n").to_string();
```

Comment:
`Cheatcode` is rather unfortunate name in scope of sncast, as we do not have cheatcodes but functions. Let's not use this jargon in messages directed to users in runtime if possible

Reply:
I disagree, the libfunc is even called `starknet::testing::cheatcode` so i don't think the naming is off

Reply:
Calling it a `function` or something else is less accurate imo but i can live with that

Reply:
I don't necessary think we should call it a `function` as well, if this message is changed smartly we can avoid both `cheatcode` and `function` I suppose

Reply:
If not `cheatcode` or `function`, any proposition? I don't see any point not to call it `function` (except that `cheatcode` would be more precise). @THenry14

Reply:
why not just `{selector} is not supported in this runtime\n` ?

Reply:
Because our error doesn't tell you the line, only the test case, and without the line, it's cryptic

Reply:
I'm not sure how an additional word "cheatcode" or "function" helps you solve this particular problem tbh. 
But I'm fine with calling this a function then